src/Whoops/Handler/Handler.php

```diff
@@ -14,13 +14,20 @@
  */
 abstract class Handler implements HandlerInterface
 {
-    /**
-     * Return constants that can be returned from Handler::handle
-     * to message the handler walker.
+    /*
+     Return constants that can be returned from Handler::handle
```

Comment:
this comment was not related to the following `DONE` constant, therefore de-phpdoc'ed as IDE help for DONE would be confusing otherwise.